sw/tests/bare-metal/hostd/idma_test.c

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include "util.h"
+#include "io.h"
+#include "printf.h"
+
+#define IDMA_BASE          0x01000000
```

Comment:
Can we move this to the [system memory map](https://github.com/pulp-platform/astral/blob/astral/sw/include/car_memory_map.h)?